rclpy/rclpy/__init__.py

```diff
@@ -49,3 +74,7 @@ def ok():
 
 def shutdown():
     return _rclpy.rclpy_shutdown()
+
+
+def get_implementation_identifier():
```

Comment:
It does return the name of an `rmw` implementation, and I guess it would be possible to have different rclpy implementations which are separate from the rmw implementation in the future. Also, I've been trying to use `rmw_implementation` everywhere. I'll add it.